deployment/charts/catalog-storage-service/values.yaml

```diff
@@ -3,7 +3,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 qe-quarkus-app:
-  replicaCount: 1
+  autoscaler:
+    minReplicas: 1
+    maxReplicas: 3
```

Comment:
From my point of view, the storage service is the only one that should keep max replicas to 1 as it's dealing with the database transactions.

Reply:
If we use a unique kafka message Key per repo/branch, for example, hash(gitRepoURL_branch)  then all request from this repo/branch is going to be processed by the same Kafka-partition (and in order), and the same consumer will handler always the same partitions, so we should not have any issue related with "concurrency/transactions". 

So, I think that this maxReplica value is fine(depends on the kafka configuration could be even higher), if the producer set as a kafka message Ke,y a unique key, per repo/branch.

Reply:
We're not using a different kafka partition per repo/branch. 
Please, use up to 1 replicas, and create a ticket to investigate the right numbers as a separate task.